migrations/migrations/20211213221929-user-create.js

```diff
@@ -2,14 +2,9 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    /**
-     * Add altering commands here.
-     *
-     * Example:
-     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
-     */
     await queryInterface.createTable('users', {
       id: {
+        primaryKey: true,
```

Comment:
Since you may not have run this migration, it is ok, but as a general rule a committed migration file must not be edited because of consistence in environments (cloud or local of other developers)

Reply:
Ok. I understand, i need to create a new migration when it happens again.